### DISCUSSION TITLE
Wii u pro controller

### DISCUSSION THREAD

**Discussion by Thecitrusak** [cathery/sys-con#192] [Controller requests]
**Description of the controller**
The description of your controller. Write down its intended usage, whether it supports USB, Bluetooth, motion controls, rumble, HD rumble, etc.
Before submitting the issue, make sure this controller is not already supported.

**Screenshot of the device**
1. Download this diagnostics applet https://files.catbox.moe/re60qo.rar
2. Run it from the homebrew menu.
3. Plug in the controller into the switch.
4. Press Y to scan and then take a screenshot.
5. Post the screenshot here and erase these steps.

**Additional context**
Add any other context about the controller request here.

**Comment by Thecitrusak** [comment]
Wii u pro controller plz

**Comment by Masamune3210** [comment]
Wii U's Pro Controller only communicates over Bluetooth, which isnt implemented yet

**Comment by t0mtee** [comment]
Wii U Pro Controller has already been requested. It was one of the first issues ever so it may be a good idea to delete / close the issue.
